com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/meta.properties

```diff
@@ -771,3 +771,12 @@ fixed_update_frequency.default = 60
 max_time_step.type = number
 max_time_step.help = If the time step is too large, it will be capped to this max value
 max_time_step.default = 0.5
+
+[fontgen]
+sdf_base_padding.type = integer
+sdf_base_padding.help = The base padding when generating sdf glyphs. [0-255]
+sdf_base_padding.default = 3
+
+sdf_edge_value.type = integer
+sdf_edge_value.help = The on edge when generating sdf glyphs. [0-255]
+sdf_edge_value.default = 190
```

Comment:
1) Should it be renamed `[font]` ?
2) If we want to support other generators in the long run, should these variables be prefixed `stbtt_` to signify that they belong to the STB TrueType settings?

Reply:
I suggest to have naming like `[stbtt_font]`. It helps avoid collisions with future implementations and helps use the same keys for parameters across different implementations (sections).